dts/arm/nxp/nxp_rt6xx.dtsi

```diff
@@ -18,6 +15,9 @@
 		peripheral: peripheral@50000000 {
 			ranges = <0x0 0x50000000 0x10000000>;
 		};
+		flexspi: spi@134000 {
+			reg = <0x5134000 0x1000>, <0x18000000 DT_SIZE_M(64)>;
+		};
```

Comment:
Reference manual has this set to `128 MB`

Reply:
I actually didn't really change this as this is what was there before. But since we are not even using this I contemplated removing the second reg (assuming you are talking about the flexspi?)